baselines/getting_started/aws/aws_check_stack/aws_account_iam_stack.tf

```diff
@@ -1,19 +1,22 @@
-## Set policy to deploy example AWS Account IAM Stack
+# AWS > Account > Stack
+# https://turbot.com/v5/mods/turbot/aws/inspect#/policy/types/accountStack
 resource "turbot_policy_setting" "aws_account_iam_stack" {
   resource = turbot_smart_folder.aws_stack.id
   type     = "tmod:@turbot/aws#/policy/types/accountStack"
   value    = "Check: Configured"
   #value    = "Enforce: Configured"
```

Comment:
remove

Reply:
not sure what should need to be removed here.